.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Version to use for the release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Version to use for further development"
+        required: true
+        default: "X.Y.Z-SNAPSHOT"
+jobs:
+  release:
+    environment: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure Git User # may be removed after https://github.com/actions/checkout/issues/13 is resolved
+        run: |
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "${{ github.actor }}"
+      - name: Set up Maven
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: 11
```

Comment:
I think Java version should be 8, some people might be building their code with JDK8 even though their Lambda runtime is set to 11.

Reply:
This is purely to run maven and upload. CI already runs both 8 and 11. `pom.xml` is set to jdk8 as well.